modules/update-upgrading-web.adoc

```diff
@@ -19,11 +19,11 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.1`.
+.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.2`.
```

Comment:
Please revert the changes in this file - I have a change in flight to address this along with some other update issues.